overlord/snapshotstate/snapshotmgr.go

```diff
@@ -371,33 +372,6 @@ func undoRestore(task *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
-func cleanupRestore(task *state.Task, _ *tomb.Tomb) error {
```

Comment:
The reference to this needs to be removed from `export_test.go`

Reply:
I force-pushed a new version, which should actually build. Given that I had to remove a unit test, I updated another one to ensure that `Cleanup()` is being called.